client/state/action-types.js

```diff
@@ -65,6 +65,11 @@ export const COUNTRY_STATES_REQUEST_SUCCESS = 'COUNTRY_STATES_REQUEST_SUCCESS';
 export const CURRENT_USER_FLAGS_RECEIVE = 'CURRENT_USER_FLAGS_RECEIVE';
 export const CURRENT_USER_ID_SET = 'CURRENT_USER_ID_SET';
 export const DESERIALIZE = 'DESERIALIZE';
+export const DIRECTLY_ASK_QUESTION = 'DIRECTLY_ASK_QUESTION';
+export const DIRECTLY_INITIALIZE = 'DIRECTLY_INITIALIZE';
+export const DIRECTLY_MAXIMIZE = 'DIRECTLY_MAXIMIZE';
+export const DIRECTLY_MINIMIZE = 'DIRECTLY_MINIMIZE';
```

Comment:
~~How do these actions get called if the user clicks the minimize button on the widget? Is that button under our control or would we need an event listener to detect when that happens?~~

Oops, I just reread the description :) You mentioned we're not keeping track of the state so there's no need for these to be called when the widget button is clicked.

If that's the case, do we need these actions? Is there any plan for the Calypso UI to trigger them, or could we simply leave them out until/if needed?